c10/core/DispatchKeySet.cpp

```diff
@@ -38,11 +74,45 @@ constexpr std::array<DispatchKey, 7> autograd_dispatch_keys {
     DispatchKey::AutogradPrivateUse1, DispatchKey::AutogradPrivateUse2,
     DispatchKey::AutogradPrivateUse3, DispatchKey::AutogradOther};
 
+constexpr std::array<DispatchKey, 30> math_dispatch_keys {
```

Comment:
Should be a comment here describing what a "math dispatch key" should be and when you should include something in here

Reply:
I'm wondering if we should generate these arrays more incrementally. For example, we could have an array for backends, an array for autograd keys, and then you'd paste them together to get math.

Reply:
This is dependent on dispatch key set union being constexpr but there isnt' any reason it shouldn't be.